tests/test_security_oauth2.py

```diff
@@ -21,18 +21,18 @@ class User(BaseModel):
     username: str
 
 
-def get_current_user(oauth_header: str = Security(reusable_oauth2)):
+def get_current_user(oauth_header: "str" = Security(reusable_oauth2)):
```

Comment:
It might be better to make this change in another file; @tiangolo if you give me some direction I can implement the tests however you'd prefer.